tests/system/conftest.py

```diff
@@ -1331,4 +1337,4 @@ def cleanup_cloud_functions(session, cloudfunctions_client, dataset_id_permanent
         # backend flakiness.
         #
         # Let's stop further clean up and leave it to later.
-        traceback.print_exception(exc)
+        traceback.print_exception(type(exc), exc, None)
```

Comment:
Moved this to https://github.com/googleapis/python-bigquery-dataframes/pull/1179